### DISCUSSION TITLE
RiMusic Websites

### DISCUSSION THREAD

**Discussion by fast4x** [fast4x/RiMusic#219] [Announcements]
Hello everyone, I purchased the domain rimusic.xyz

The purpose is to have a landing page on an external domain, combined with 
- the RiMusic Github.com page
- the RiMusic F-Droid page, if all goes well. Currently F-Droid is considering the request for inclusion of RiMusic.

To create the landing page, I would like to use a free template called Venus that I have already inserted into the Venus folder of the domain.

https://rimusic.xyz/venus/

I like the general look, I already imagine it with the screenshots and the features of RiMusic.

See if you like it, if you propose alternative solutions. 

I would also like to create a new repository specifically for the template that we can work on together.

Thank you always for your support.

**Comment by ikanakova** [comment]
I'd like to help with anything. I'm quite good in html (including Bootstrap) and I know also a bit of php and javascript.
I think @OrangeZXZ and @senolfndk will come up with some nice design that will match the app.

**Reply by senolfndk** [reply]
You can benefit from these or get inspired, I don't know if they will be useful to you. 
https://m3.material.io/ https://fluent2.microsoft.design/

**Reply by ikanakova** [reply]
> You can benefit from these or get inspired, I don't know if they will be useful to you.
https://m3.material.io/ https://fluent2.microsoft.design/

I'll take a look at it.

**Comment by OrangeZXZ** [comment]
I like the venus look but i think it dosen't really fit rimusic

In my opinion the site should look a lot like the app that way the user already has a feel from the app before installing

Im not a designer but i can try to do an site design if I have time

**Reply by OrangeZXZ** [reply]
Also what is the default rimusic font? is it Rubick?

**Reply by ikanakova** [reply]
> Also what is the default rimusic font? is it Rubick?

If you mean the logo font, it's Varela Round. As for the actual font in the app, I'm not sure, but it's probably Poppins.

**Reply by fast4x** [reply]
Confirm, is Poppins in the app

**Comment by senolfndk** [comment]
You can benefit from these or get inspired, I don't know if they will be useful to you. 
https://m3.material.io/ https://fluent2.microsoft.design/

> Herhangi bir konuda yardım etmek isterim. Html konusunda oldukça iyiyim (Bootstrap dahil) ve biraz da php ve javascript biliyorum. Bence@OrangeZXZVe@senolfndkUygulamayla eşleşecek güzel bir tasarım bulacağız.

**Comment by OrangeZXZ** [comment]
> I like the venus look but i think it dosen't really fit rimusic
> 
> In my opinion the site should look a lot like the app that way the user already has a feel from the app before installing
> 
> Im not a designer but i can try to do an site design if I have time

I did a quick design for the desktop site with weird proportions
![image](https://github.com/fast4x/RiMusic/assets/104271365/e11447b8-d53d-4edc-ac18-19b9eaeba2cc)

**Reply by fast4x** [reply]
It seems to me too much like the app, it almost doesn’t look like a presentation website. It’s just my idea eh, maybe I’m wrong.

**Reply by OrangeZXZ** [reply]
> It seems to me too much like the app, it almost doesn’t look like a presentation website. It’s just my idea eh, maybe I’m wrong.
> 

you are right i need to remake it this was just an idea i had

**Reply by fast4x** [reply]
I’ll try an experiment from your drawing.

**Reply by fast4x** [reply]
> > It seems to me too much like the app, it almost doesn’t look like a presentation website. It’s just my idea eh, maybe I’m wrong.
> > 
> 
> you are right i need to remake it this was just an idea i had

Try starting with Venus...

**Reply by OrangeZXZ** [reply]
> Try starting with Venus...

I made a Venus design for the "Home" on desktop
![image](https://github.com/fast4x/RiMusic/assets/104271365/fadbb0f2-36cd-4dae-9481-ff6163c1baf7)

I used black hole site for reference
https://sangwan5688.github.io

**Reply by fast4x** [reply]
I love it! Congratulations!

**Reply by fast4x** [reply]
But I can’t use their site, it’s not published as open source

**Reply by OrangeZXZ** [reply]
> But I can’t use their site, it’s not published as open source

i can try to do something similar in the RiMusic UI Style

**Reply by ikanakova** [reply]
> i can try to do something similar in the RiMusic UI Style

I think it could redo the Venus design in black or something on Bootstrap, because the image you posted is much better than the Black Hole page.

**Reply by OrangeZXZ** [reply]
> I think it could redo the Venus design in black or something on Bootstrap, because the image you posted is much better than the Black Hole page.

How about this?
![image](https://github.com/fast4x/RiMusic/assets/104271365/f0adbcd7-b9d5-480a-8ce1-47a71b8f77a2)

**Comment by ikanakova** [comment]
> > I think it could redo the Venus design in black or something on Bootstrap, because the image you posted is much better than the Black Hole page.
> 
> How about this? ![image](https://private-user-images.githubusercontent.com/104271365/290918349-f0adbcd7-b9d5-480a-8ce1-47a71b8f77a2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI3MjY5OTUsIm5iZiI6MTcwMjcyNjY5NSwicGF0aCI6Ii8xMDQyNzEzNjUvMjkwOTE4MzQ5LWYwYWRiY2Q3LWI5ZDUtNDgwYS04Y2UxLTQ3YTcxYjhmNzdhMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxNlQxMTM4MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NTE5YjI4ODllMGI0NTZmZjZkYmViZTUzYzU2MTVlNTYzYTBiM2UyOGZlY2MxYjllMDdkODliNWEzYWI3OTE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.qJMCMXYZQXc32Tf6SQ3-RA3n6MWNGP7oFjZFh46Qxio)

I meant this picture:
<img src="https://private-user-images.githubusercontent.com/104271365/290774188-fadbb0f2-36cd-4dae-9481-ff6163c1baf7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI3MjY5OTUsIm5iZiI6MTcwMjcyNjY5NSwicGF0aCI6Ii8xMDQyNzEzNjUvMjkwNzc0MTg4LWZhZGJiMGYyLTM2Y2QtNGRhZS05NDgxLWZmNjE2M2MxYmFmNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxNlQxMTM4MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2E2ZDg2NzhhNzhiMzZmM2Y4NDY5M2ExZmQyZDI5M2M5MDM2ZWU4OTQzNWY2YmU3ZGFmMjc4ZWRiZDkwMjYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Iyc4OLHL7-7SPsm_D5OHGxe6BXvlFDaZsVzNNCel5j8">

**Comment by fast4x** [comment]
From now, RiMusic Is available on
GitHub, IzzyOnDroid, F-Droid and rimusic.xyz

Thanks at all

**Comment by fast4x** [comment]
Updated rimusic website with new logo and icon.
